### DISCUSSION TITLE
Can't use the essentialsX economy system

### DISCUSSION THREAD

**Discussion by hightechatri** [PotatoCraft-Studio/QuickShop-Reremake#478] [Q&A] [answered]
I am an administrator of a minecraft server,using paper in version 1.20.4.I used essentialsX to manage my server.
![image](https://github.com/user-attachments/assets/95183d30-7c2b-4534-a94e-55a76eded1ee)
As we can see,essentialsX already has a economy system while quickshop still telling me economy system not loaded
![image](https://github.com/user-attachments/assets/362bb957-74ef-4012-a8fb-5aa7a2fbb4d5)
How can I use the essentialsX economy system while using quickshop?

**Comment by YuanYuanOwO** [comment]
请上传插件列表以及服务器日志

**Reply by hightechatri** [reply]
![image](https://github.com/user-attachments/assets/c51b7e3f-d951-4d8a-80b1-3263ebcf714b)
这是插件的列表 我查阅了服务器日志后未发现服务端有报错信息如图
![image](https://github.com/user-attachments/assets/26537fd4-c01f-4259-97bf-4f0b599581cf)

**Reply by hightechatri** [reply]
顺带一提 我在十几分钟前将服务器升级到了1.21paper端

**Reply by YuanYuanOwO** [reply, marked as answer]
> ![image](https://github.com/user-attachments/assets/c51b7e3f-d951-4d8a-80b1-3263ebcf714b)
> 这是插件的列表 我查阅了服务器日志后未发现服务端有报错信息如图
> ![image](https://github.com/user-attachments/assets/26537fd4-c01f-4259-97bf-4f0b599581cf)
> 

1.QuickShop-Hikari的相关问题不要在此仓库询问
2.您的 Vault 未安装，QuickShop不会工作。

**Reply by hightechatri** [reply]
好的 谢谢您的回复与对我提问错仓库的谅解，祝您生活愉快
